src/Symfony/Bundle/TwigBundle/Resources/config/console.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
+
+        <!-- Alias for BC -->
+        <service id="twig.command.debug" alias="console.command.symfony_bundle_twigbundle_command_debugcommand" />
+        <service id="console.command.symfony_bundle_twigbundle_command_debugcommand" class="Symfony\Bridge\Twig\Command\DebugCommand">
+            <argument type="service" id="twig"/>
+            <tag name="console.command" command="debug:twig" />
+        </service>
```

Comment:
Right.. i also realized i deprecated the alias, which probably wont work. We can deprecate aliases. Although current XML is happily parsed.

So we basically need to preserve the old service (not as an alias). It will point to the old class and its definitions is deprecated.

I believe that will conflict with auto discovery (deprecated will override), perhaps make a special case there. But i havent followed the code path for this approach yet.. maybe it just works :)